command/genesis/polybft_params.go

```diff
@@ -46,7 +46,7 @@ const (
 	defaultCheckpointInterval       = uint64(900)
 	defaultWithdrawalWaitPeriod     = uint64(1)
 	defaultVotingDelay              = "10"
-	defaultVotingPeriod             = "20"
+	defaultVotingPeriod             = "10000"
 	defaultVoteProposalThreshold    = "1000"
```

Comment:
WDYT about adding comment in which units these are expressed? For example, these are expressed in blocks if I'm not mistaken. Would be awesome to add such comments for each default parameter value.

Reply:
Done: https://github.com/0xPolygon/polygon-edge/pull/1761/commits/064b2429b4cc1f6bcc19002ad7bf3cd79713d0e6